torch/_inductor/sizevars.py

```diff
@@ -346,7 +346,7 @@ def size_hint(self, expr: Expr) -> int:
         try:
             return int(out)
         except Exception:
-            log.warning("failed on: %s", out)
+            log.debug("failed on: %s", out)
```

Comment:
Sooo is word on the street that `size_hint` legitimately can raise exceptions that we're going to catch later? In that case we probably should just stop doing the logging here and do some exception chaining instead.

Reply:
You introduced this logging and it was triggering way too often:-) Usual size hints are ok, strides goof sometimes (pretty often, in fact, as we've learned due to logging), but strides also catch the exception.